layouts/partials/components/carousel.html

```diff
@@ -0,0 +1,42 @@
+<section>
+	<div id="tinyslider-carousel">
+		{{ range sort $.Params.carousel.images "weight" }}
+		<div class="item">
+			<div>
+				<img class="img-responsive" src="{{ .image }}" alt="{{ .description }}" />
```

Comment:
Use `utlis/get-image.html` may help you to get the url of image. Because it can handle images in static/assets folders or post folder or external links.

You can use something like `{{ $img := partial "utils/get-image" (dict "context" . "url" url-of-image "keyword" "*url-of-image*")}}`. url and keyword look similar because get-image will use keyword to find images in post folder and url will try to find images in assets/static/external link.

Reply:
Can you give me a bit of example how to do that with variables? I tried few of the following, yet nothing worked:

```
{{ $img := partial "utils/get-image" (dict "context" . "url" .image "keyword" | printf "*%s*" .image)}}
				<img class="img-responsive" src="{{ $img }}" alt="{{ .description }}" />
```
```
{{ $img := partial "utils/get-image" (dict "context" . "url" . "keyword" | printf "*%s*" .)}}
				<img class="img-responsive" src="{{ $img }}" />
```

I feel like it should be a simple answer :)